src/redux/books/APICall.js

```diff
@@ -0,0 +1,33 @@
+import axios from 'axios';
```

Comment:
- [ ] According to the project requirement you should use [fetch](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch) or [axios](https://axios-http.com/docs/intro) with [createAsyncThunk](https://redux-toolkit.js.org/api/createAsyncThunk) for making API calls. Please fix it in order to make your project works as expected :+1: